packages/itwinui-css/src/bridge.scss

```diff
@@ -9,16 +9,37 @@
 
   --iui-color-background: var(--stratakit-color-bg-page-base);
   --iui-color-background-disabled: var(--stratakit-color-bg-glow-on-surface-disabled);
-  --iui-color-background-zebra: var(--iui-color-background); /* TODO: Use proper value when added */
-  --iui-color-background-hover: var(--iui-color-background);
+  --iui-color-background-zebra: color-mix(
+    in oklch,
+    var(--stratakit-color-bg-page-base),
+    var(--stratakit-color-bg-page-zebra)
+  );
+  --iui-color-background-hover: color-mix(
+    in oklch,
+    var(--iui-color-background) 100%,
+    var(--stratakit-color-glow-hue) var(--stratakit-color-bg-glow-base-hover-\%)
+  );
   --iui-color-background-backdrop: var(--stratakit-color-bg-page-base);
-  --iui-color-background-backdrop-hover: var(--iui-color-background-backdrop);
+  --iui-color-background-backdrop-hover: color-mix(
+    in oklch,
+    var(--iui-color-background-backdrop) 100%,
+    var(--stratakit-color-glow-hue) var(--stratakit-color-bg-glow-base-hover-\%)
+  );
   --iui-color-background-accent: var(--stratakit-color-bg-accent-base);
-  --iui-color-background-accent-hover: var(--iui-color-background-accent);
-  --iui-color-background-accent-muted: var(--stratakit-color-bg-glow-on-surface-accent-pressed);
-
-  --iui-color-border-foreground: var(--stratakit-color-border-mono-muted);
-  --iui-color-border-foreground-hover: var(--iui-color-border-foreground);
+  --iui-color-background-accent-hover: color-mix(
+    in oklch,
+    var(--iui-color-background-accent) 100%,
+    var(--stratakit-color-glow-hue) var(--stratakit-color-bg-glow-base-hover-\%)
+  );
+  --iui-color-background-accent-muted: var(--stratakit-color-bg-accent-muted);
```

Comment:
Noticed an issue while testing #2642.

`--stratakit-color-bg-accent-muted` looks too saturated. I think it's supposed to use `--stratakit-color-bg-glow-on-surface-accent-pressed` (based on the new ListItem/TreeItem components).

One difference between the two variables is transparency, which is something we'll need to account for at component level if necessary (as in the case of FileUpload).